packages/default-icons/src/figma-icons/Legend16.tsx

```diff
@@ -0,0 +1,16 @@
+import { SVGProps, memo } from 'react';
+
+const Legend16 = (props: SVGProps<SVGSVGElement>) => (
+  <svg width={16} height={16} viewBox="0 0 16 16" fill="none" {...props}>
+    <path d="M7.44504 12.9339H14.2524" stroke="#D2D5D8" strokeLinecap="square" strokeLinejoin="round" />
```

Comment:
do you need it in this color, or it should we currentColor here?

Reply:
fixed everywhere